### DISCUSSION TITLE
How to define a 3rd party JS URL as breeze component?

### DISCUSSION THREAD

**Discussion by sandipansaha** [breezefront/community#187] [1. Help]
Hi @vovayatsyuk 
Could you please guide me on how could we define a 3rd party JS URL as a breeze component through breeze_default.xml?
Example,
https://unpkg.com/@adobe/magento-storefront-events-sdk/dist/index.js
Thanks

**Comment by vovayatsyuk** [comment]
Breeze doesn't differ from any other website when writing js. You need to include this script on the page, wait until it's loaded, and use the exposed variables/functions. The script you want to add creates global `magentoStorefrontEvents` variable.

Examples:

1. Classic approach

```
<script src="https://unpkg.com/@adobe/magento-storefront-events-sdk@1.5.0/dist/index.js"></script>
<script>
alert(window.magentoStorefrontEvents)
</script>
```

2. Import inside component after user interaction

```
$.widget('...', {
  create: function () {
    $.lazy(() => require(['https://unpkg.com/@adobe/magento-storefront-events-sdk@1.5.0/dist/index.js'], () => {
        alert(window.magentoStorefrontEvents);
    }));
  }
})
```
